server/channels/app/platform/license.go

```diff
@@ -353,7 +353,7 @@ func (ps *PlatformService) GenerateRenewalToken(expiration time.Duration) (strin
 	claims := &JWTClaims{
 		LicenseID:   license.Id,
 		ActiveUsers: activeUsers,
-		StandardClaims: jwt.StandardClaims{
+		RegisteredClaims: jwt.RegisteredClaims{
 			ExpiresAt: expirationTime.Unix(),
```

Comment:
```suggestion
			ExpiresAt: jwt.NewNumericDate(expirationTime),
```

The type of the `ExpiresAt` field changed compared to the same field in the old `StandardClaims` struct.

Reply:
Thank you for checking and changing the code.